docs/source/markdown/podman-image-sign.1.md

```diff
@@ -36,6 +36,10 @@ Store the signatures in the specified directory.  Default: /var/lib/containers/s
 
 Override the default identity of the signature.
 
+#### **--auth-file**=*file
+
+Override the default auth file of the signature
```

Comment:
Can you make this more descriptive, please? This doesn't make it very clear what the flag does

Reply:
At the very least, you need a period at the end.

```suggestion
Override the default auth file of the signature.
```

Also, this should be `authfile`, not `auth-file`.  I think you can just grab the verbiage from http://docs.podman.io/en/latest/markdown/podman-pull.1.html#authfile-path or some of the other authfile options in Podman and put it in here too.